docs/integrations-guides/rollup-guides/orbit/deployment.md

```diff
@@ -21,53 +21,68 @@ Arbitrum nodes communicate with EigenDA via the proxy for secure communication a
+| action version | network | address |
+|---------|---------|---------|
+| [eigenda-v2.1.3](https://github.com/Layr-Labs/orbit-actions/blob/63ba07bbaa849117d2074ccd3c90c2628c58b36d/scripts/foundry/contract-upgrades/eigenda-v2.1.3/README.md)  | Ethereum Holesky | [0x31C2C3dd9C47049a975DFaE804C886333CDCF9BB](https://holesky.etherscan.io/address/0x31C2C3dd9C47049a975DFaE804C886333CDCF9BB#code) | 
+| [eigenda-v2.1.3](https://github.com/Layr-Labs/orbit-actions/blob/63ba07bbaa849117d2074ccd3c90c2628c58b36d/scripts/foundry/contract-upgrades/eigenda-v2.1.3/README.md)  | Arbitrum Sepolia | [0xc5d5c28F94c95D851a42Ff97f1886f8af505BD04](https://sepolia.arbiscan.io/address/0xc5d5c28F94c95D851a42Ff97f1886f8af505BD04#code) | 
+| [eigenda-v2.1.3](https://github.com/Layr-Labs/orbit-actions/blob/63ba07bbaa849117d2074ccd3c90c2628c58b36d/scripts/foundry/contract-upgrades/eigenda-v2.1.3/README.md)  | Base Sepolia     | [0xbda5cff32ce4b2a1e312439e21d1b163d48b0936](https://sepolia.basescan.org/address/0xbda5cff32ce4b2a1e312439e21d1b163d48b0936) | [0xA0735f425792eCcDA1b8f50ceB36B05635c77D63](https://sepolia.basescan.org/address/0xa0735f425792eccda1b8f50ceb36b05635c77d63)
 
-5. Once your rollup is deployed, you will be redirected to review the rollup configurations. Make sure to download the zip file containing your chain and node config files, and store them somewhere handy.
 
-6. Go to the next page and follow the instructions there to set up a nitro sequencer using the downloaded config files to power the rollup. If you've completed these steps successfully, you are now running an Arbitrum Orbit rollup that uses EigenDA!
-
-## Token Bridge
+## How to deploy a Rollup on Testnet using our UI
 
-The Arbitrum token bridge can be enabled to support L1 to/from L2 bridging of ERC-20 assets. Since the token bridge is a wrapper on-top of the existing L1 to/from L2 native bridge, there are no changes necessary to enable it. Additionally, the [existing](https://docs.arbitrum.io/build-decentralized-apps/reference/contract-addresses#token-bridge-smart-contracts) token bridge creators maintained by Offchain labs can be leveraged to deploy token bridges on-top of existing inboxes integrated EigenDA.
+While you can interact with the deployed Rollup creator directly, we recommend using our [orbit chain deployment portal](https://orbit.eigenda.xyz/) to deploy a rollup for a friendlier devx and easy-to-use configs. Currently, the UI only supports testnets for:
+- Ethereum Holesky
+- Arbitrum Sepolia
 
 
 ### Troubleshooting
+If your nitro setup script node encounters a warning `error getting latest batch count: no contract code at given address`, you should first verify that:
+    - **(1)** That the `SequencerInbox` entry in your `/config/orbitSetupScriptConfig` maps to a successfully deployed contract
+    - **(2)** Your RPC provider is sufficiently reliably. Transient errors are common when leveraging free and public RPC providers
```

Comment:
"is sufficiently reliably" -> "is sufficiently reliable"

Reply:
0cbc508970a0d7f6d09f81618ff6f2fe544bcbf9